creator-node/src/routes/files.js

```diff
@@ -192,13 +192,34 @@ module.exports = function (app) {
       if (req.query.filename) {
         res.setHeader('Content-Disposition', contentDisposition(req.query.filename))
```

Comment:
@vicky-g this set header should be done regardless of whether you stream from ipfs or fs. this should be pulled out of this try catch

Reply:
the set header call is done prior to both ipfs and fs so we don't have to worry about the header not being set :)

Reply:
even though it looks like this code will work, i agree with dheeraj that logically this code should be outside the try-catch